ddp-workspace/projects/ddp-rarex/src/app/components/activities-list/activities-list.component.ts

```diff
@@ -28,6 +28,7 @@ export class ActivitiesListComponent {
     ActivityCodes.HealthAndDevelopment,
     ActivityCodes.QualityOfLife,
     ActivityCodes.ParentalQualityOfLife,
```

Comment:
Sorry, I didn't realize there's stuff on the frontend. In latest changes, we split parental QOL into 2 separate activities, so now we have 3. What changes are needed on frontend?